crates/rpc-types-beacon/src/lib.rs

```diff
@@ -2,12 +2,13 @@
 //!
 //! Provides all relevant types for the various RPC endpoints, grouped by namespace.
 
+#![doc = include_str!("../README.md")]
```

Comment:
is this a drive-by fix? don't think we touched the beacon crate in this PR

Reply:
Drive-by fix for consistency, removed the comment block for now, updated the README